### PR TITLE
Fix width and height typing and add auto-shipit

### PR DESCRIPTION
**Pull request by cmarin001**
This pull request addresses essential enhancements within our application's development ecosystem, focusing on the correction of width and height property typings across components and the integration of an auto-shipit feature for automated deployments. Aimed at refining the codebase and optimizing deployment processes, these updates ensure accurate type definitions for UI elements and a more efficient, automated release workflow.

### CODE REVIEW

File: src/Grid/stories/stories.styles.ts [hunk 1]
```diff
@@ -5,12 +5,12 @@ const StyledSquare = styled.div`
...
   justify-content: center;
   align-items: center;
-  background-color: ${inube.color.surface.gray.regular};
+  background-color: ${inube.palette.neutral.N30};
   border-radius: 4px;
   padding: 3px;
   height: 50px;
   width: 100%;
-  color: ${inube.color.text.light.regular};
+  color: ${inube.palette.neutral.N10};
```
Comment (wfercanas) [inline src/Grid/stories/stories.styles.ts:13]: Let's change this color to a darker one, could be a N100.

File: src/Grid/index.tsx
Comment (wfercanas) [inline src/Grid/index.tsx:1]: The build process is including `react/jsx-runtime`

**Review by wfercanas — CHANGES_REQUESTED** [review]

File: src/Grid/stories/stories.styles.ts (hunk 1 above)
Comment (cmarin001) [reply src/Grid/stories/stories.styles.ts]: solved

File: src/Grid/index.tsx
Comment (cmarin001) [reply src/Grid/index.tsx:1]: solved

**Review by wfercanas — APPROVED** [review]

**Comment by JanuarAsprilla** [comment]
image
run the `npm run build` command and check to fix the issues you are having

**Comment by JanuarAsprilla** [comment]
In the `package.json` you need to implement the guideline, which is to place the `peerDependencies` and the packages that go inside these

**Review by JanuarAsprilla — APPROVED** [review]

**Review by wfercanas — APPROVED** [review]

**Pull request merged**